src/gettingstarted/own-code/service-configuration.md

```diff
@@ -0,0 +1,69 @@
+# Import your own code
+
+## Configure services
+
+In the previous step, you created a collection of empty configuration files that have given your project the following structure:
+
+```.
+├── .platform
+│   ├── routes.yaml
+│   └── services.yaml
+├── .platform.app.yaml
+└── < application code >
+```
+
+Now you will need to include information that will tell Platform.sh how you want your application to connect to its [services](/configuration/services.md). An example `.platform/services.yaml` will always look something like this:
+
+`.platform/services.yaml`
+
+{%- codetabs name="Python", type="py", url="https://raw.githubusercontent.com/platformsh/template-python3/master/.platform/services.yaml" -%}
+
+{% language name="PHP", type="php", url="https://raw.githubusercontent.com/platformsh/template-php/master/.platform/services.yaml" -%}
+
+{% language name="Go", type="go", url="https://raw.githubusercontent.com/platformsh/template-golang/master/.platform/services.yaml" -%}
+
+{% language name="Node.js", type="js", url="https://raw.githubusercontent.com/platformsh/platformsh-example-nodejs/master/.platform/services.yaml" -%}
+
+{%- endcodetabs %}
+
+If you're application does not use any services at this point, you can leave it blank, but it must exist in your repository to run on Platform.sh. If your application does use a database or other services, you can configure them with the following attributes:
+
+* `name`: Provide a name for the service, so long as it is alphanumeric. If your application requires multiple services of the same type (i.e., three MySQL databases), make sure to give them different names so that your data from one service is never overwritten by another (i.e., `mysqldb1`, `mysqldb2`, `mysqldb3`).
+
+* `type`: This specifies the service type and its version using the format `type:version`. Consult the table below that lists all Platform.sh maintained services, along with their `type` and supported `version`s. The links will take you to each service's dedicated page in the documentation.
+
+<div>
+  <table id="servicesTable" border="1">
+  <tbody></tbody>
+  </table>
+</div>
+
+<script>
+makeImagesTable("services", "supported", "servicesTable");
+</script>
+
+* `disk`: The `disk` attribute configures the amount of persistent disk that will be allocated between all of your services. Projects by default are allocated 5 GB (5120 MB), and that space can be distributed across all of your services.
```

Comment:
Add: "Note that not all services require disk space. If you specify a `disk` attribute for a service that doesn't use it, like Redis, you will receive an error when trying to push your changes."